### PR TITLE
Add Logger

### PR DESCRIPTION
**Pull request by kwokyto**

### CODE REVIEW

File: src/main/java/seedu/fridgefriend/food/Fridge.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package seedu.fridgefriend.food;
+
+import java.util.ArrayList;
+
+public class Fridge {
+    private static ArrayList<Food> fridge = new ArrayList<>();
```
Comment (Vinci-Hu) [inline src/main/java/seedu/fridgefriend/food/Fridge.java:6]: Why is this class level?

File: src/main/java/seedu/fridgefriend/utilities/Logger.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
+package seedu.fridgefriend.utilities;
+
+import java.util.logging.Level;
+import java.lang.Throwable;
+
```
Comment (Vinci-Hu) [inline src/main/java/seedu/fridgefriend/utilities/Logger.java:5]: Should explain what Logger class does here?

**Review by Vinci-Hu — APPROVED** [review]
Code and OOP generally LGTM, if there is no bugs or failed tests can merge.

File: src/main/java/seedu/fridgefriend/food/Fridge.java (hunk 1 above)
Comment (kwokyto) [reply src/main/java/seedu/fridgefriend/food/Fridge.java:6]: so we can statically access this without having to pass a Fridge object through methods

File: src/main/java/seedu/fridgefriend/utilities/Logger.java (hunk 2 above)
Comment (kwokyto) [reply src/main/java/seedu/fridgefriend/utilities/Logger.java:5]: will add javadoc comments

**Review by SimJJ96 — APPROVED** [review]
LGTM

**Comment by Leeyp** [comment]
Thanks for adding the Logger. LGTM.

**Review by Leeyp — APPROVED** [review]
Thanks for adding the Logger. LGTM!

**Pull request merged**